INSTALL.txt

```diff
@@ -63,7 +63,7 @@ Here is the content of this archive:
 From there on, follow these instructions:
 
 1. Upload at least index.php and prestashop.zip on your web server.
-2. From your web browser, go to the folder where index.php and prestashop.zip have been uploaded and browse index.html. The Zip archive should unzip automatically.
+2. From your web browser, go to the folder where index.php and prestashop.zip have been uploaded. The Zip archive should unzip automatically.
```

Comment:
```suggestion
2. From your web browser, go to the folder where index.php and prestashop.zip have been uploaded and browse index.html. The Zip archive should unzip automatically.
```

On 2nd thought ... 🤔 how is it ? better or not ? I'm 50/50 on it

Reply:
i say the same in the issue https://github.com/PrestaShop/PrestaShop/issues/21463

Heads or tails ? :rofl:

Reply:
`rand(1,2)` => my php returned 2